### DISCUSSION TITLE
How can I give experience(XP) out on mobile?

### DISCUSSION THREAD

**Discussion by Gavinotfound** [SoftbearStudios/mk48#231] [Q&A] [answered]
As the title suggests, is there a command or button that could give out XP on mobile? Making upgrading oil rigs as an PC-exclusive feature is surely not right.

**Comment by finnbear** [comment, marked as answer]
Hello,

The feature you are searching for doesn't currently exist (as it would have required additional UI complexity such as a toggle button for dispensing coins). Here are some workarounds:
1. find already-upgraded oil rigs
  - from old bases
  - in the Arctic biome at the top of the world
2. ask a member of your team to upgrade them for you
